lib/dynamic_config/dcdo.rb

```diff
@@ -55,6 +55,7 @@ def frontend_config
       'student-email-post-enabled': DCDO.get('student-email-post-enabled', false),
       'progress-v2-metadata-enabled': DCDO.get('progress-v2-metadata-enabled', false),
       'show-updated-lms-content': DCDO.get('show-updated-lms-content', false),
+      'curriculum-launch-2024': DCDO.get('curriculum-launch-2024', false),
```

Comment:
If you merge in staging you can remove this since I added it in this PR: https://github.com/code-dot-org/code-dot-org/pull/59271